README.md

```diff
@@ -1 +1,59 @@
-# ToDoList
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do List
+> This web application provides users a convenient way to keep track of their daily routine works. Users can add and remove tasks from the list of tasks they curate on the website.
+
+
+![SCREENSHOT](./assets/todo.GIF)
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+
+## Clone and review it
+
+Run the following commands to clone and run it.
+
+### To clone the repository
+
+  `git@github.com:HAMMAS-SALEEM/ToDoList.git`
```

Comment:
- [ ] [Required] I suggest you specify all the commands for cloning your repository here. In that way, every developer can easily clone the repository without having to try to remember the exact code for that. What about we say: `git clone git@github.com:HAMMAS-SALEEM/ToDoList.git` instead? What do you think?